upup/models/cloudup/resources/addons/authentication.aws/k8s-1.10.yaml.template

```diff
@@ -37,9 +37,16 @@ spec:
         image: {{ or .Authentication.Aws.Image "602401143452.dkr.ecr.us-west-2.amazonaws.com/amazon/aws-iam-authenticator:v0.4.0" }}
         args:
         - server
+        {{- if or (not .Authentication.Aws.BackendMode) (contains "MountedFile" .Authentication.Aws.BackendMode) }}
```

Comment:
We have two version of the the addon manifest (two template files) one for k8s 1.10+ and one for k8s 1.12+. Given that we only updated the 1.12+ manifest to authenticator 0.5.1: https://github.com/kubernetes/kops/commit/eae3fd8fc3efa212d80afc444e26e0323157b1d3#diff-49ab9104184d27772708fb0ecd00e0aa we dont need to add BackendMode functionality to the 1.10+ manifest, so we can probably remove all the changes to this file.